contracts/TopShot.cdc

```diff
@@ -369,7 +372,7 @@ pub contract TopShot: NonFungibleToken {
         //
         // Returns: The NFT that was minted
         // 
-        pub fun mintMoment(playID: UInt32): @NFT {
+        access(NFTMinter) fun mintMoment(playID: UInt32): @NFT {
```

Comment:
I'm a little confused why you have a minter entitlement but don't have entitlements for any of the other methods? This never has a public capability, right? So why the need for a single entitlement here?